README.md

```diff
@@ -1,17 +1,53 @@
 # Cartography
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE)![Version](https://img.shields.io/nuget/v/Cartography.DynamicMap?style=flat-square)![Downloads](https://img.shields.io/nuget/dt/Cartography.DynamicMap?style=flat-square)   
 
-Nventive solution for mobile app map.
+Cartography provides you 3 map related libraries for IOS, Android and windows using native map of each platform.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+## **Cornerstones**
+ - Add native map to your app.
+ - If needed, Google map can be use on all platform.
+ - The solution generate three independant nuget.
 
 ## Getting Started
 
 ### **Samples**
 - Clone project a sample is available.
 - For seeing samples, build and install app with VS on the desire device (Android, IOS, or UWP)
 
-&nbsp;
-## **DynamicMap**
+### API Key (not needed for mapService)
+- For Google map (Android), you need to [create your Api Key](https://developers.google.com/maps/documentation/javascript/get-api-key)
+- For Bing map, you should need a [Bing Api Key](https://learn.microsoft.com/en-us/bingmaps/getting-started/bing-maps-dev-center-help/getting-a-bing-maps-key) but work fine without it in sample.
```

Comment:
This is confusing.
> You should need (...)
> (...) but work fine without it in sample.

Is it required or not? What's the impact of having it vs not having it?